### ISSUE TITLE
Why Yolov5s-p34 has better recognition results than Yolov5s？

### ISSUE THREAD

**Issue by Hehezcr** [ultralytics/yolov5#10203]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Because my dataset is a single-target small target, I trained with YOLOv5s-p34.yaml, and the final result was that YOLOv5s-p34 was nearly 4% higher than YOLOv5s' Recall, Precision, and Map, but I don't know why it became higher. Because YOLOv5s only removes P5, it should only reduce the model size. My training set is 2600 and more than 40000 labels, and my validation set is more than 300 and 6000 labels. Hope for an explanation.

**Comment by glenn-jocher** [comment]
@Hehezcr this has been answered elsewhere, P34 will work better if your dataset is all small objects.
